all-actors/src/main/java/org/sunbird/cert/actor/CertificateGeneratorActor.java

```diff
@@ -111,25 +109,26 @@ private String upload(String certFileName) {
 			//return storageParams.upload(System.getenv(JsonKey.CONTAINER_NAME), "/", file, false);
 
 			File file = FileUtils.getFile("conf/certificate/"+certFileName);
+			logger.info("CertificateGeneratorActor:upload:container name got from env is: ".concat(System.getenv(JsonKey.CONTAINER_NAME)));
 			return AzureFileUtility.uploadFile(System.getenv(JsonKey.CONTAINER_NAME),file);
 		}catch (Exception ex) {
 			logger.info("CertificateGeneratorActor:upload: Exception occurred while uploading certificate.",ex);
 		}
 		return "";
 	}
 
-	private HashMap<String,String> getProperties(){
-		// properties need to populate from env
+	private HashMap<String,String> populatePropertiesMap(Request request){
+		String rootOrgId=(String)((Map)request.get(JsonKey.CERTIFICATE)).get(JsonKey.ROOT_ORG_ID);
```

Comment:
In cert generate request are we asking for rootOrgId as well

Reply:
Yes for Badge  we need rootOrgId as well as BatchId